build.gradle.kts

```diff
@@ -7,22 +10,32 @@ import org.jetbrains.intellij.IntelliJPlugin
 import org.jetbrains.intellij.IntelliJPluginExtension
 import org.jetbrains.intellij.tasks.*
 import org.jetbrains.intellij.tasks.RunPluginVerifierTask.FailureLevel.*
+import se.ascp.gradle.GradleVersionsFilterPlugin
 
 plugins {
   checkstyle
   `java-library`
   scala
```

Comment:
TBH it's weird that both `scala` and `kotlin-dsl` plugins are shipped with Gradle, but Kotlin plugin isn't 🤯